docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,492 @@
+<div id="toggle-button">
+    {% include section-header.html name="toggle-button" version=page.versions.toggle-button %}
+
+    <p>The Toggle Button is a button that can be toggled/pressed on or off. It is an atomic piece of the Toggle Button Group.</p>
+    
+    <p>Titles are required and other elements, such as subtitles and images can be included as well.</p>
+
+    <p>These buttons should <strong>NOT</strong> contain structural elements, such as headings or lists.</p>
+
+    <p>For detailed requirements and usage details, please review the <a href="https://ebay.gitbook.io/mindpatterns/input/toggle-button">eBay MIND Pattern for Toggle Button</a></p>
```

Comment:
I'm just going to remove this for now, as what we have in MIND doesn't perfectly line up (I will get them both in better sync later)
```suggestion
```